src/com/dre/brewery/listeners/Compat1_9.java

```diff
@@ -0,0 +1,31 @@
+package com.dre.brewery.listeners;
+
+import com.dre.brewery.Brew;
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerItemConsumeEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.PotionMeta;
+
+// Workaround to remove unwanted potion effects
+public class Compat1_9 implements Listener {
+
+	@EventHandler
+	public void onPlayerDrink(PlayerItemConsumeEvent event) {
+		ItemStack item = event.getItem();
+		Brew brew = Brew.get(item);
+		if (brew == null) {
+			return;
+		}
+
+		if (item.getType() == Material.POTION) {
+			try {
+				PotionMeta meta = (PotionMeta) item.getItemMeta();
```

Comment:
@Sataniel98 Do you remember what other approaches you followed? I have a few ideas forming but if you've already traveled those paths I won't do it a second time :) .

The first thought is simply creating a new potion with nothing as base effect (water bottle?) and the brew's effects applied to it, and using the setItem method of the consume event to replace what is being drunk. Then, by contract the _real_ consumable isn't removed, so using the PlayerInventory object we could remove it from the inventory.

If you've tried that and it didn't work ... well, I'll have to keep thinking on it :) Might be able to just remove the potion entirely, just delete it from inventory, and apply the effects from the brew directly to the Player, instead of constructing a replacement potion.